README.rst

```diff
@@ -27,24 +27,51 @@ Welcome to nose2
 
 **Note**: As of 0.7.0 we no longer support 2.6, 3.2, or 3.3. We also removed ``nose2.compat``.
 
-nose2 is the next generation of nicer testing for Python, based
-on the plugins branch of unittest2. nose2 aims to improve on nose by:
+``nose2`` aims to improve on nose by:
 
  * providing a better plugin api
  * being easier for users to configure
  * simplifying internal interfaces and processes
  * supporting Python 2 and 3 from the same codebase, without translation
  * encouraging greater community involvement in its development
 
-In service of some those goals, some features of nose *will not* be
-supported in nose2. See `differences`_ for a thorough rundown.
+In service of some those goals, some features of ``nose`` *are not*
+supported in ``nose2``. See `differences`_ for a thorough rundown.
 
-In time -- once unittest2 supports plugins -- nose2 should be able to
-become just a collection of plugins and configuration defaults. For
-now, it provides a plugin api similar to the one in the unittest2
-plugins branch, and overrides various unittest2 objects.
+Workflow
+--------
 
-You are witnesses at the new birth of nose, mark 2. Hope you enjoy our
-new direction!
+If you want to make contributions, you can use the ``Makefile`` to get started
+quickly and easily::
+
+    # no prior setup necessary!
```

Comment:
Well.... You need to have a supported version of python installed. Also maybe add a note here that (maybe obviously) tox can only test locally against versions of python you have installed, but we'll only merge if tests pass for all supported versions.